utils/sample_qc.py

```diff
@@ -363,99 +363,114 @@ def infer_sex(
     return (sex_ht)
 
 
+def get_ploidy_cutoffs(
+    ht: hl.Table, 
+    f_stat_cutoff: float, 
+    normal_ploidy_cutoff: int = 5, 
+    aneuploidy_cutoff: int = 6
+) -> tuple:
+    """
+    Gets chromosome X and Y ploidy cutoffs for XY and XX samples. Note this assumes the input hail Table has the fields f_stat, chrX_ploidy, and chrY_ploidy.
+    Returns a tuple of sex chromosome ploidy cutoffs: ([x_ploidy_cutoffs], [y_ploidy_cutoffs]).
+    x_ploidy_cutoffs: [upper cutoff for single X, (lower cutoff for double X, upper cutoff for double X), lower cutoff for triple X]
+    y_ploidy_cutoffs: [lower cutoff for single Y, upper cutoff for single Y, lower cutoff for double Y]
```

Comment:
what do you think of using `((lower cutoff for single Y, upper cutoff for single Y), lower cutoff for double Y)` for consistency' sake? so that similar to X each element of the returned value corresponds to a specific ploidy?
Otherwise, I think we should change `x_ploidy_cutoffs` to: `(upper cutoff for single X, (lower cutoff for double X, upper cutoff for double X), lower cutoff for triple X)`